README.md

```diff
@@ -9,6 +9,20 @@ flow between a PSP and the Payment Node.
 
 This repository is part of the [Digital Citizenship project](https://teamdigitale.governo.it/it/projects/cittadinanza-digitale.htm).
 
+In this version was made a customization on autogenerated types to import the package in italia-pagopa-proxy. Importo types was modified from 
+
+```
+importoSingoloVersamento: "minInclusive" | "maxInclusive" | "fractionDigits" | "totalDigits";
+
+```
+to 
+
+```
+importoSingoloVersamento: number;
```

Comment:
@LsKoder I though you said we were going to use a more restrictive type for amounts